packages/amplify-category-api/src/provider-utils/awscloudformation/cfn-api-artifact-handler.ts

```diff
@@ -59,6 +59,7 @@ class CfnApiArtifactHandler implements ApiArtifactHandler {
     const stacksDirectoryPath = path.join(resourceDir, stacksDirName);
     if (!fs.existsSync(stacksDirectoryPath)) {
       fs.mkdirSync(stacksDirectoryPath);
+      fs.copyFileSync(path.join(rootAssetDir, 'resolver-readme', 'README.md'), `${resolverDirectoryPath}/README.md`);
```

Comment:
nit use `path.join()` for the destination path as well

Reply:
Done